### ISSUE TITLE
Jargon Is Good, Actually

### ISSUE THREAD

**Issue by dreeves** [beeminder/blog#7]
This comes up a lot and I keep pointing people to the Onomastic Interlude section of https://blog.beeminder.com/brl/ so maybe that wants to be its own blog post.

See also:
https://forum.beeminder.com/t/does-beeminder-use-too-much-jargon/6702

**Comment by dreeves** [comment]
This just came up today with "let's get that qualsified / qualificated". Adam was like "um, we already have the word 'qualified' right there in the dictionary" and my contention is that it needs to _not_ be in the dictionary because otherwise the listener thinks they understand. This is the Mark Twain thing: it's not what you don't know that gets you in trouble, it's what you know that ain't so.

**Comment by adamwolf** [comment]
The word qualified is used in this sense already!  A perfectly valid use of
the verb "to qualify" is to make a series of assertions and compare them
against a process or an artifact.

"Has the new PCB been qualified yet?" is something I have heard at least 50
times.

If i had a time machine, the first thing I would do is see a show from
REM's '89 world tour.  The second would have been to go back and loudly
protest against "quals".

**Comment by dreeves** [comment]
Ah, I didn't know that sense of "qualify". (Though I now see that larsivi mentioned it in our [discord debate](https://forum.beeminder.com/t/quals-vs-specs-vs-tests/10050).) Of course that just makes me like the term "quals" even more. But obviously I'm missing something if you'd rather undo that terminology change than kill baby Hitler. Maybe RubyMine is a significant source of the pain?

**Comment by dreeves** [comment]
Another case study I like: https://github.com/beeminder/beeminder/issues/3412#issuecomment-1242488249 (about the term "fawx" instead of "init")

**Comment by dreeves** [comment]
Fun debate today:

random musing: "cybernetics" is, i think, kind of a fancy synonym for "control systems". sometimes i try to awkwardly turn "control systems" into an adjective or adverb like "I'm beeminding this control-systems-style". so would it be too pretentious to instead say "I'm beeminding this cybernetically"? if even i am rolling my eyes a little at that then probably it's too much... but maybe it'd grow on us?

[then we talked about whether "thermostatic" could be used metaphorically to convey the idea. eventually adam pointed out that "homeostatic" works quite well here.]

and later in the discussion:

ah, yes, cue my side rant about not accepting "people won't understand" as a reason not to use le mot juste! i like treating the dictionary as the bright line. if it's in the dictionary it's fair game. (fun fact: i had to dig through my anki to find the perfect word for "the perfect word". i remembered it existed but not what it was. and it is in fact in the (english) dictionary. https://www.merriam-webster.com/dictionary/mot+juste )

so, yeah, my philosophy is that people absolutely will understand, by looking the word up. that's what literate people do. i am probably overly strident about this though.

(also, ironically, i'm not entirely confident i'm using "mot juste" correctly there!)

**Comment by dreeves** [comment]
Pithification: If you dejargon the jargon, people _don't know that they don't know_ what you're talking about. 

Consider "I updated on that" vs "I bayesian-updated on that". 

But does that example work? In the first sentence, "updated" is absolutely a jargon term and the problem is how easy it is to sail past it thinking it's just awkward phrasing or a typo for "I updated that". A reader who doesn't know this definition of "updated on" is likely to not notice that they don't know it.

In that example you want either a clearly jargon-y jargon word like "bayesian-updated" or you want a longer sentence that conveys the idea of an updated belief distribution conditional on new evidence.

Which is all to say that you want to know your audience and use the language that optimizes communication with them. Beeminder is (for now) making the strategic decision to lean in on [targeting nerds](https://blog.beeminder.com/nerds "Strategy Memo: Beeminder Is For Nerds (2020)"), including word nerds. But we can definitely use reminders of when we go too far!

**Comment by dreeves** [comment]
more notes from discord and slack:

1. don't sound pretentious. i see "bayesian-updated" as convenient shorthand for "i learned new information and shifted my opinion accordingly, not necessarily a 180-degree changing of my mind, but something on the spectrum" but the point is well taken that it's worth considering if you need all that nuance or if you can just be a normal human and say "this is causing me to potentially reconsider" or something. it can be tricky because it's common for listeners to filter out hedge words like "potentially" and remember you as having said you changed your mind, full stop. but, yeah, you're very correct about sounding pretentious. something for me to work on for sure!

2. defining jargon with tooltips is :brain: :100:. of course, not everything is html. or if, say, you're talking in the forum, it could be html but you're not realistically gonna take the time to linkify everything. if the jargon is searchable/googlable, that's some portion of the benefit. but, yeah, more tooltips and/or links to help docs is really smart.

> It talks about things like the interoceptively signaled allostatic costs of social interactions with predicted prediction error

:flushed:
but this is such good jargon! words i can easily search for and they're based on latin so even though i don't speak latin i have hints for remembering them. ie, there's decent evocativity, to use my term from messymatters.com/nominology

now i'm reminded of https://twitter.com/touchmoonflower/status/1614790331857502209 which i think @adamwolf pointed me to and i think that if you want to give an ordinary word a technical definition (like freud with "it", "I", and "over-I") then borrowing from another language is really perfect. like how freud's translator went with the latin words for "it", "I", and "over-I": ego, id, and superego. the alternative is relying on italicization or just a lot of context and zero searchability.

the author of that twitter thread castigates freud's translator ("decided that such an important concept needed to be made more obscure and technical sounding") but i'm pretty convinced it was a great choice. of course I use latin for our bug reports so, of course i think that.

we started talking about this in the discord too and poisson/sean pointed how how confusing physics can be about using jargon that doesn't look like jargon. eg, 

* big bang (this one's probably fine though)
* work, power, energy, force (especially "strong force" and "weak force")
* acceleration and jerk and snap and crackle and pop? (no, these are probably great)
* c for the speed of light?

i think math has some pretty bad examples, like giving "almost surely" and "almost everywhere" an insanely specific technical meaning. maybe that should be my core point in all of this: don't delude yourself that you've dejargoned jargon by just co-opting/overloading simple words. that makes it worse. you may sound less pretentious but at the cost of deceiving the listener into thinking they've understood you when they haven't!

---

sanity check on the thesis i'm groping my way toward: y'know how math has an insanely specific technical definition of "almost surely" and "almost everywhere"? what if mathematicians started using latin for those. "pen certum" and "pen universum" or whatever the proper latin would be.

i claim that that's a marked improvement. sure, it sounds more pretentious but it's far friendlier to the reader who's not deceived into thinking they've understood something when they haven't. they can just google those terms and get right up to speed.

(not that you can just unilaterally switch to latin, but, like, if it became an actual thing with a wikipedia page, that would be better. yes, we already have https://en.wikipedia.org/wiki/Almost_surely but the point is you don't know that you don't know what it means so you don't ever find that page.)

[bluastelo disagrees at this point]

i may concede the point that anyone who can understand the above wikipedia page also understands that "almost surely" is being used in a technical sense, ie, as jargon. but... maybe there's something else going on that i haven't put my finger on. it now occurs to me that lawyers have a ton of jargon, some of which is latin and some of which is overloaded simple english phrases. do we have intuitions about whether one category is better or worse? my intuition is that the latin is helpful. it's a big handy flag marking something as being used in a technical sense. it mostly works either way but you have to be careful with the english versions.

(i think i need an example now, if i want this to be at all convincing, where using jargon that doesn't look like jargon causes legit confusion.)

oh yeah, @Sean had an example: https://en.wikipedia.org/wiki/Large_deviations_theory
to be clear, @Sean specifically claimed it caused him a bunch of confusion because he didn't know [that] he didn't know what "large deviation" meant 

---

we do sometimes purge jargon! like "yellow brick road" and all the stuff with "lanes". and we changed "mercy" to "post-derail respite", not sure if that counts.
we used to call ratcheting "retro-ratchet" which was superfluously jargon-y

BREWSTER: as a new beeminder user, the jargon is not off putting. the product is clearly not designed for a general audience. so when I was reading blog posts about stuff like the akrasia horizon, I had already bought into the mental hurdle of learning new concepts/phrasing. IMO the product's marketing filters out people who would have an issue with jargon. given the strategic decision you mentioned to lean in on nerds, I don't think dejargoning the jargon should be a high priority

---

(changing "max safety buffer" to "auto-ratchet" was a good example of valuable jargon, i think. we want to convey that it's a specific feature doing a specific technical thing. not just talking about the most safety buffer you have or have had or something. oh yeah, plus searchability.)

---

BLUASTELO: I tend to agree with Feynman about jargon: if you _have_ to use it, you (generic, not personal) don't know what you're talking about.

**Comment by dreeves** [comment]
more examples of annoyingly colloquial-sounding jargon:
* common knowledge (sounds like "thing everyone knows" but sometimes you need to talk about the specific technical thing where everyone knows something, everyone knows everyone knows it, everyone knows everyone knows everyone ... ad infinitum)
* efficient (sounds like some kind of cold utilitarianism at the expense of aesthetics but i want it as shorthand for "that which maximizes aggregate utility")
* utility (same problem -- i want shorthand for "that which is the object of your deepest, truest preferences; anything you'd make sacrifices for")

those three are a perennial thorn in my side. sometimes i say "socially efficient" to try to clarify the version of efficiency i'm talking about but i think that's worse. it sounds vaguely nazi-adjacent or something. i think it absolutely has the problem where someone who doesn't have any background in econ or game theory will sail past an instance of "socially efficient" imagining that communication has happened.

and did we mention "field"? an annoyingly hard-to-google word having totally different technical meanings in math and in physics.

**Comment by dreeves** [comment]
Consider "epicenter". It's Greek for "on the center" and you might think it's a pointlessly jargony version of "center" but it'd tend to be more awkward, when talking about earthquakes, to always have to disambiguate uses of "center" that referred to specifically to being directly above the point at which the strain energy in the rock is first released.

Switching languages or inventing a portmanteau can add a lot of clarity compared to overloading simple existing words with technical meanings.

**Comment by dreeves** [comment]
we were talking today about rational and irrational numbers and i wanted to say "let's try this with some actual numbers (as opposed to variables)" but i accidentally said "let's try this with some real numbers" and that was totally confusing and ambiguous (cuz in math, "real numbers" means "a complete ordered field such that..." and holy cow, "complete", "ordered", and "field" all have this same problem!) and i felt vindicated on this "jargon works best when pretentiously borrowed from other languages" thesis of mine.

i'm only some fraction serious! but definitely a nonzero fraction! and i get that when fields (not that kind of "field") do this it does mostly work fine and you can almost (not that kind of "almost") always distinguish, say, "real" and "real" in context. maybe i have a few different theses here:

1. jargon is not just about intimidating outsiders -- it's necessary shorthand.
2. using simple words as jargon isn't actually de-jargoning. 
3. simple words as jargon is confusing.

**Comment by dreeves** [comment]
More examples:

* "gissue" vs "issue"
* "gol" vs "goal"
* "adam" vs "hardware_nerd_friend_in_mn_with_two_boys"

**Comment by dreeves** [comment]
thinking more about this after someone mentioned doing something "in test", referring to a dev environment. i realize how silly it sounds to say "don't use english" and i think what i really mean is is that overloading terms yields ambiguity and confusion.

**Comment by dreeves** [comment]
"The disadvantage of using normal words that normal people use is that these are clearly 'magic words' and not just plain descriptions of things." --DD

**Comment by dreeves** [comment]
Speaking of "normal words", today in the Snake Eyes Discord someone accidentally referred to a "normal distribution" to mean ... something other than a Gaussian? I think? The fact that I'm not sure is what proves the point: Turning everyday words into jargon is confusing!

**Comment by dreeves** [comment]
Sort of related: in our family discord we name the channels as cute portmanteaus such that you can always refer to the channel name without having to disambiguate it. eg, "can you put that in mathpotatoes?" rather than "can you put that in the 'math' channel on discord?".

I thought we had made a similar point in https://blog.beeminder.com/goalnames but apparently not. It's now my biggest reason for cute Beeminder goalnames, to be able to say things like "I have a steparoo beemergency" or just "i have steparoo to do" instead of "I have a beemergency on my 'steps' goal".

**Comment by dreeves** [comment]
me: blah blah designing #TARE tags to replace the odometer reset feature
clive: ...or you could just go with #RESET, or something else that non-native English speakers might be able to recognise more easily?

**Comment by dreeves** [comment]
just had a doctor's appointment with cantor for some knee pain and the doctor was trying to ask where the pain is, asking about "below the kneecap" and we're like "below as in under or... um, beneath, or... no, none of these words actually clarify" and i'm like "can you just speak latin??"

(turns out "subpatellar" means below the kneecap as in further into the body, as opposed to "inferior to the kneecap" meaning closer to the foot)

**Comment by dreeves** [comment]
me today in the family discord:

according to the internet, "set" is the word in english with the most definitions. so of course mathematicians use it for the most fundamental concept in all of maths. i have a whole diatribe about how annoying mathematicians are about this, turning everyday words into technical jargon. it's so bad! because a newbie doesn't know what they don't know, y'know? like they hear you talking about "real numbers" and presume you just mean "actual numbers" as opposed to hypothetical numbers. or specific numbers as opposed to variables. once you're inculcated it's fine but it would, ironically, be so much clearer if the math term for the reals were some fancy latin thing like "continuata" or "fluxorum". you see that, immediately know you don't know what it means, look it up, et voila, now you know. with "real" you have to see it enough times to catch on that it's being used in a technical sense and then construct a disambiguating search query like "real number math".

in conclusion, jargon is fine and valuable among experts. avoiding jargon is also great, when you want to be more accessible. but using everyday words _as_ jargon is the worst of both worlds.

**Comment by bsoule** [comment]
<img width="900" height="900" alt="Image" src="https://github.com/user-attachments/assets/c9489f46-7037-42d9-9981-9770f1794897" />
